webplugin/js/app/voice/index.js

```diff
@@ -0,0 +1,70 @@
+class Voice {
+    _VOICE_PLATFORM_API_URL = 'https://api.elevenlabs.io';
+    _VOICE_PLATFORM_API_KEY = 'sk_5af5c862dd8c87860ece549378c67628233440a018ac2e12';
```

Comment:
can this be a problem as this is on customer's website so we don't have control of restricting in any domain?
better to download this key from server, so that if we ever to have change this api key, we can change it from server